src/main/com/fulcrologic/rad/rendering/semantic_ui/report.cljc

```diff
@@ -257,12 +257,12 @@
                                (dom/a {:onClick (fn [evt]
                                                   (evt/stop-propagation! evt)
                                                   (report/sort-rows! report-instance column))}
-                                 (str label)
```

Comment:
Oh boy...technically a breaking change. If someone used a keyword, say, for a label, then their code would have worked before and will crash now. I think I'm going to let this slide because having DOM elements as a label is more important in this case, and it was a genuine oversight. BUT, needs to be listed in the commit message as a BREAKING CHANGE.